src/test/java/io/divolte/server/filesinks/hdfs/FileFlusherLocalHdfsTest.java

```diff
@@ -133,15 +132,16 @@ public void shouldRollFilesWithSimpleStrategy() throws IOException, InterruptedE
 
         flusher.cleanup();
 
-        final MutableInt count = new MutableInt(0);
-        Files.walk(tempPublishDir)
-             .filter((p) -> p.toString().endsWith(".avro"))
-             .forEach((p) -> {
-                 verifyAvroFile(records, schema, p);
-                 count.increment();
-             });
+        final long verifiedFileCount =
+            Files.walk(tempPublishDir)
+                 .filter(p -> p.toString().endsWith(".avro"))
+                 .filter(p -> {
+                     verifyAvroFile(records, schema, p);
+                     return true;
+                 })
+                 .count();
```

Comment:
But of course. It must have been harsh times that defeated my abstinence of such impurity.